tailwind.config.js

```diff
@@ -22,7 +22,34 @@ module.exports = {
             background: 'hsl(var(--background))',
             foreground: 'hsl(var(--foreground))',
             cursor: 'hsl(var(--cursor))',
-
+            lightcorrect: '#2ECC71',
+            lightwrong: '#E74C3C',
+            darkcorrect: '#27AE60',
+            darkwrong: '#E74C3C',
+            lightorangecorrect: '#228B22',
+            lightorangewrong: '#DC143C',
```

Comment:
My God no. 
This must have been so much work for you to do here. wow...
so inside this file, just add the variables, say
`
 correct_char: 'hsl(var(--correct_char))'
 wrong_char: 'hsl(var(--wrong_char))'
`
then head into `global.css` and add this colors to the themes there.